### PR TITLE
Update DK, New Yaml

### PR DESCRIPTION
**Pull request by sherryw94**

### CODE REVIEW

File: public/data/travel/DK.yml [hunk 1]
```diff
@@ -1,46 +1,104 @@
 ---
 sources:
   domestic:
-  - "[Domestic travel is not restricted](https://www.visitdenmark.com/denmark/travel-essentials/safe-travel-denmark)
-    and it is not mandatory to wear a mask."
+  - 'Denmark has now implemented [Phase 4](https://www.themayor.eu/en/denmark-tightens-coronavirus-restrictions) of its lockdown removal plan.'
+  - 'As of August 22, 2020, all passengers aged 12 or more must wear [face masks or shields on all means of public transport](https://politi.dk/en/-/media/mediefiler/corona/genaabning/face-masks-or-shields-required-on-all-forms-of-public-transport-from-22-august-2020.pdf?la=en&hash=3438F39C3B8F72E5FA2BE5BC514308FF091C910C), on ferries and in taxis. Customer-facing staff must also wear face masks or shields.'
+  - 'Educational institutions and indoor sports and recreational facilities have [reopened](https://politi.dk/en/coronavirus-in-denmark/controlled-reopening-of-denmark).'
+  - 'Venues such as nightclubs and discotheques (venues with few seats, with a dance floor) must [stay closed](https://politi.dk/en/coronavirus-in-denmark/controlled-reopening-of-denmark) until October 31, 2020.'
```
Comment (jannwong) [inline public/data/travel/DK.yml:7]: night clubs - 2 words
discotheques change to disco

File: public/data/travel/DK.yml [hunk 2]
```diff
@@ -1,46 +1,104 @@
...
   inbound:
-  - "[Denmark’s borders are open](https://www.visitdenmark.com/denmark/travel-essentials/safe-travel-denmark)
-    to most European countries (except Portugal and Sweden), as well as Australia,
-    Canada, Japan, New Zealand, South Korea, and Thailand from June 27, 2020."
-  - Tourists from the approved countries must show documentation of a [valid booking
-    for a minimum of 6 days on entry.](https://www.visitdenmark.com/denmark/travel-essentials/safe-travel-denmark)
+  - 'Travelers from EU Member States, including San Marino and Vatican City, Schengen countries, United Kingdom, Australia, Canada, Georgia, Japan, New Zealand, South Korea, Thailand, Tunisia and Uruguay are [allowed entry](https://politi.dk/en/coronavirus-in-denmark/travelling-in-or-out-of-denmark/is-my-country-open-or-banned) into Denmark.'
+  - 'However, travelers from Andorra, Belgium, Croatia, France, Luxembourg, Malta, Romania, Spain and Monaco are only allowed entry for ["worthy purpose"](https://reopen.europa.eu/en/map/DNK) only, not for tourism.'
```
Comment (jannwong) [inline public/data/travel/DK.yml:15]: change "worthy purpose" to essential travel

File: public/data/travel/DK.yml [hunk 3]
```diff
@@ -1,46 +1,104 @@
...
+  - CA
+  - GE
+  - JP
+  - NZ
+  - KR
+  - TH
+  - TN
+  - UY
+  outbound: 'yes'
+  inbound_restricted:
```
Comment (jannwong) [inline public/data/travel/DK.yml:69]: when you use inbound_allowed, you don't need to input inbound_restricted. the map logic is such that any countries that are not in inbound_allowed, will be restricted

**Review by jannwong — CHANGES_REQUESTED** [review]
inline comments

File: public/data/travel/DK.yml (hunk 3 above)
Comment (sherryw94) [reply public/data/travel/DK.yml:69]: ah ok!

**Review by jannwong — APPROVED** [review]

**Pull request merged**